### PR TITLE
webdriver api's explained

### PR DESCRIPTION
**Pull request by manoj9788**

### CODE REVIEW

File: 02_Getting_Started_with_WebDriver.md [hunk 1]
```diff
@@ -98,10 +98,54 @@ Acting on the AUT
...
-String name = "Charles";
-driver.findElement(By.name("name")).sendKeys(name);
+driver.findElement(By.Id("some id")).click();
+```
+Using SendKeys API to handle a type in WebDriver:
+```java
+String name = "selenium";
+driver.findElement(By.name("locatorByName")).sendKeys(name);
+```
+Using JavaScript in WebDriver:
```
Comment (andreastt) [inline 02_Getting_Started_with_WebDriver.md:110]: Executing arbitrary JS in the browser should not be covered in the getting started guide IMO.  Perhaps we should have an additional section in the WebDriver proper chapter?

File: 02_Getting_Started_with_WebDriver.md [hunk 2]
```diff
@@ -98,10 +98,54 @@ Acting on the AUT
...
+	(new Actions(driver)).dragAndDrop(sourceElement, targetElement).perform();
+	```
+	* handling mouse hover in WebDriver:
+	```java
+	Actions builder = new Actions(driver);
+	WebElement tagElement = driver.findElement(By.id("some id"));
+	builder.moveToElement(tagElement).build().perform();
+	```
+	
+Using Select API to select a value from a drop down:
```
Comment (andreastt) [inline 02_Getting_Started_with_WebDriver.md:129]: The Select utilities is a fairly advanced topic.  Should we perhaps cover this in the WebDriver or the best practices chapter?

File: 02_Getting_Started_with_WebDriver.md (hunk 1 above)
Comment (manoj9788) [inline 02_Getting_Started_with_WebDriver.md:110]: yes, introducing Js here will be a bad idea and instead we could add this up elsewhere.

you can remove this.

File: 02_Getting_Started_with_WebDriver.md (hunk 2 above)
Comment (manoj9788) [inline 02_Getting_Started_with_WebDriver.md:129]: Select is one of the commonly used API in WebDriver and I believe it should be here. :)

**Comment by andreastt** [comment]
Committed as 4407add0858d35a616a70ec9e1edd24671387eb1.

**Pull request closed without merging**